core/trino-main/src/main/java/io/trino/execution/CreateTableTask.java

```diff
@@ -253,7 +263,11 @@ else if (element instanceof LikeClause) {
         Set<String> specifiedPropertyKeys = statement.getProperties().stream()
                 .map(property -> property.getName().getValue())
                 .collect(toImmutableSet());
-        Map<String, Object> finalProperties = combineProperties(specifiedPropertyKeys, properties, inheritedProperties);
+
+        Map<String, Object> finalProperties = combineProperties(
```

Comment:
You can leave the line as before. It is not that long and this syntactic change doesn't make in this case the code more readable.